navi/src/test/java/com/trello/navi2/rx/RxNaviActivityTest.java

```diff
@@ -21,331 +20,296 @@
   private final NaviEmitter emitter = NaviEmitter.createActivityEmitter();
 
   @Test public void observeCreate() {
-    TestSubscriber<Bundle> testSubscriber = new TestSubscriber<>();
-    Subscription subscription = RxNavi.observe(emitter, Event.CREATE).subscribe(testSubscriber);
-    testSubscriber.assertNoValues();
+    TestObserver<Bundle> testObserver = new TestObserver<>();
+    RxNavi.observe(emitter, Event.CREATE).subscribe(testObserver);
+    testObserver.assertNoValues();
 
     Bundle bundle = new Bundle();
     emitter.onCreate(bundle);
-    subscription.unsubscribe();
+    testObserver.dispose();
     emitter.onCreate(bundle);
 
-    testSubscriber.assertValue(bundle);
-    testSubscriber.assertNoTerminalEvent();
-    testSubscriber.assertUnsubscribed();
+    testObserver.assertValue(bundle);
+    testObserver.assertNotTerminated();
   }
 
   @Test public void observeCreatePersistable() {
-    TestSubscriber<BundleBundle> testSubscriber = new TestSubscriber<>();
-    Subscription subscription =
-        RxNavi.observe(emitter, Event.CREATE_PERSISTABLE).subscribe(testSubscriber);
-    testSubscriber.assertNoValues();
+    TestObserver<BundleBundle> testObserver = new TestObserver<>();
+    RxNavi.observe(emitter, Event.CREATE_PERSISTABLE).subscribe(testObserver);
```

Comment:
Funny, I was just reading about this and deciding whether I wanted to switch or not. I'm not coming up with any compelling reason to go through the effort, since all the code is already there.